candle-nn/src/loss.rs

```diff
@@ -1,12 +1,13 @@
+//! Loss functions.
 use candle::{Result, Tensor};
 
 /// The negative log likelihood loss.
 ///
 /// Arguments
 ///
-/// * [inp]: The input tensor of dimensions `N, C` where `N` is the batch size and `C` the number
+/// * `inp`: The input tensor of dimensions `N, C` where `N` is the batch size and `C` the number
 ///          of categories. This is expected to contain log probabilities.
-/// * [target]: The ground truth labels as a tensor of u32 of dimension `N`.
+/// * `target`: The ground truth labels as a tensor of u32 of dimension `N`.
```

Comment:
These were all unresolved links:

```
warning: unresolved link to `target`
  --> /home/ivor/Documents/Code/rust/playground_candle/candle/candle-nn/src/loss.rs:36:8
   |
36 | /// * [target]: The ground truth labels as a tensor of u32 of dimension `N`.
   |        ^^^^^^ no item named `target` in scope
   |
   = help: to escape `[` and `]` characters, add '\' before them like `\[` or `\]`
```

There is no current official way to document function arguments, I followed https://github.com/rust-lang/rust/issues/57525.

Rendered:

![image](https://github.com/huggingface/candle/assets/1732289/deca0cfd-1c12-40c0-adbc-e1bf918cdca7)